.prettierignore

```diff
@@ -8,7 +8,10 @@ jest.preset.js
 *.MD
 lib/shared/bucketing-assembly-script/
 .yarn
-.husky
+.huskyx
 .verdaccio
 node_modules
 **/.next/**
+.github/**
+nx.json
+lerna.json
```

Comment:
why do these need to be ignored?

Reply:
They're being errored against - even though they're correctly formatted.